table/block_based/index_builder.h

```diff
@@ -51,13 +48,13 @@ class IndexBuilder {
     Slice index_block_contents;
     std::unordered_map<std::string, Slice> meta_blocks;
   };
-  explicit IndexBuilder(const InternalKeyComparator* comparator, size_t ts_sz,
-                        bool persist_user_defined_timestamps)
+  IndexBuilder(const InternalKeyComparator* comparator, size_t ts_sz,
```

Comment:
Would you mind help removing the `explicit` qualifier from the constructors for some other IndexBuilder subclasses in this file?